### PR TITLE
Add test routes

### PR DESCRIPTION
**Pull request by imurchie**
Add the `/test/...` routes that are used in our hybrid/web tests.

### CODE REVIEW

File: lib/static.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    res.cookie('guineacookie3', 'cant access this', {
+      domain: '.blargimarg.com',
+      path: '/'
+    });
+    res.send(getTemplate('guinea-pig.html')(params));
+  }, delay);
+}
+
+function welcome (req, res) {
+  var params = { message: 'Let\'s browse!' };
```
Comment (jlipps) [inline lib/static.js]: no need for surrounding spaces here

Comment (jlipps) [inline lib/static.js]: also, var!!

**Comment by jlipps** [comment]
small comments then LGTM

**Comment by imurchie** [comment]
There's always a `var` somewhere. Fixed.

**Pull request merged**